starter-code/MyLinkedList.java

```diff
@@ -12,15 +12,46 @@ public int getSize() {
 	}
 
 	public boolean remove(int index) {
-		//to-do
+		if (index > size) {
+			return false;
+		} else {
+			Node<T> headNode = head;
+			Node<T> nextNode;
+			for (int i = 0; i < index; i++) {
+				headNode = headNode.getNext();
+			}
+			nextNode = headNode.getNext();
+			headNode.setNext(nextNode);
+			size--;
+			return true;
+		}
 	}
 
 	public T get(int index) {
-		//to-do
+		if (index > size) {
+			return null;
+		} else {
+			Node<T> headNode = head;
+			for (int i = 0; i < index; i++) {
+				headNode = headNode.getNext();
+			}
+			return (T) headNode.getData();
```

Comment:
Looks good! Don't forget to check if `index` is a bogus value like a negative number - in that case you can just return null.